components/lib/calendar/calendar.d.ts

```diff
@@ -566,6 +566,10 @@ interface CalendarBaseProps {
      * @defaultValue false
      */
     disabled?: boolean | undefined;
+    /**
+     * Specify the date that cannot be selected.
+     */
+    disabledDate?: ((day: number, month: number, year: number) => boolean) | undefined;
```

Comment:
Why not just use a `disabledDates` array with a single valyue?

Reply:
Sorry I don't understand Can you give me an example? thanks!

Reply:
Well it looks like you are trying to disable a "single" day right?

Why not just use `disabledDates?: Date[] | undefined;`

So you can just do...

```js
const disabledDay = new Date(2023, 10, 15);
const disabledDates = [disabledDay]; // array of 1 day

<Calendar disabledDates={disabledDates} />
```

Reply:
Yes I was just trying to prevent adding new props.